src/oncall/auth/modules/ldap_example.py

```diff
@@ -1,33 +1,67 @@
 # Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
 # See LICENSE in the project root for license information.
 import ldap
+import os
+import logging
 
+logger = logging.getLogger(__name__)
 
 class Authenticator:
     def __init__(self, config):
         if config.get('debug'):
             self.authenticate = self.debug_auth
             return
+        self.authenticate = self.ldap_auth
+
+        if 'ldap_cert_path' in config:
+            self.cert_path = config['ldap_cert_path']
+            if not os.access(self.cert_path, os.R_OK):
+                logger.error("Failed to read certificate")
+                return False
+
+        if 'ldap_bind_user' in config:
+            self.bind_user = config['ldap_bind_user']
```

Comment:
Makes me a little nervous having these attributes only sometimes exist. Can we set to None instead to avoid sneaky AttributeErrors?